tools/envoy_headersplit/headersplit.py

```diff
@@ -0,0 +1,378 @@
+# !/usr/bin/env python3
+# Lint as: python3
+"""
+This python script can dividing monolithic mock headers
```

Comment:
Nit: This seems a very short line width BTW. I suggest wrapping at Envoy standard length of 100 for all comments (and code).